src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useState } from "react";
+import { useHistory } from "react-router-dom";
+import styled from "styled-components";
+
+import { saveFacility } from "../database/index";
+import InputButton from "../design-system/InputButton";
+import InputText from "../design-system/InputText";
+import ChartArea from "../impact-dashboard/ChartArea";
+import ImpactProjectionTable from "../impact-dashboard/ImpactProjectionTableContainer";
+import MitigationInformation from "../impact-dashboard/MitigationInformation";
+import useModel from "../impact-dashboard/useModel";
+import { FacilityContext } from "./FacilityContext";
+import FacilityInformationSection from "./FacilityInformationSection";
+import LocaleInformationSection from "./LocaleInformationSection";
+import RateOfSpreadSection from "./RateOfSpreadSection";
+import { Facility } from "./types";
+
+const FacilityInputFormDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+const LeftColumn = styled.div`
+  margin: 20px;
+`;
+const RightColumn = styled.div`
+  margin: 20px;
+`;
+
+// TODO add section header tooltips
+// TODO add summary at bottom of Locale Information
+const FacilityInputForm: React.FC = () => {
+  const { facility } = useContext(FacilityContext);
+  const history = useHistory();
+  const [facilityName, setFacilityName] = useState(facility?.name);
+  const model = useModel();
+
+  const save = () => {
+    saveFacility({
+      id: facility?.id,
+      name: facilityName,
+      modelInputs: JSON.parse(JSON.stringify(model))[0],
```

Comment:
I'm not crazy about this JSON stuff, but this is what I needed to do to get the inputs in a shape such that Firestore would accept them

Reply:
Agreed! I filed https://github.com/Recidiviz/covid19-dashboard/issues/144 to look into it later.

Reply:
Note: This line doesn't work with IE11. Browsers with native Map support serialize Maps as `{}`, browsers without native Map support try to serialize the Map polyfill object and get a circular reference error.

Reply:
@szhu did you come up with a replacement for this in your IE11 / build work, or is this broken as of yet?

Reply:
@jessex yes, sorry the discussion for this has been very split across a few threads. Just going to link to the threads here so we can't have to find them again:
https://github.com/Recidiviz/covid19-dashboard/pull/225/files#r415322977
https://github.com/Recidiviz/covid19-dashboard/pull/225/files#r415377061

Ideally I would have like to make this its own PR, but I needed to fix this to make IE11 work. Maybe I should actually just move this to its own PR so we can discuss there?